packages/flame_oxygen/lib/src/component.dart

```diff
@@ -5,8 +5,11 @@ import 'package:flame/particles.dart';
 import 'package:flame/sprite.dart';
 import 'package:oxygen/oxygen.dart';
 
+export 'package:flame/particles.dart' show Anchor, Particle;
+
+part 'component/anchor_component.dart';
 part 'component/angle_component.dart';
 part 'component/particle_component.dart';
 part 'component/position_component.dart';
 part 'component/size_component.dart';
-part 'component/sprite_component.dart';
\ No newline at end of file
+part 'component/sprite_component.dart';
```

Comment:
It is recommended to avoid `part` and use export instead.
https://dart.dev/guides/libraries/create-library-packages